### PR TITLE
Feat formule

### PR DESCRIPTION
**Pull request by Valentin460**

### CODE REVIEW

File: src/Controller/SubscriptionController.php [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            return $this->redirectToRoute('app_subscription_index', [], Response::HTTP_SEE_OTHER);
+        }
+
+        return $this->render('subscription/new.html.twig', [
+            'subscription' => $subscription,
+            'form' => $form,
+        ]);
+    }
+
+    #[Route('/{id}', name: 'app_subscription_show', methods: ['GET'])]
```
Comment (horotopia) [inline src/Controller/SubscriptionController.php:51]: Does it works ?

File: src/Controller/SubscriptionController.php [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+            return $this->redirectToRoute('app_subscription_index', [], Response::HTTP_SEE_OTHER);
+        }
+
+        return $this->render('subscription/edit.html.twig', [
+            'subscription' => $subscription,
+            'form' => $form,
+        ]);
+    }
+
+    #[Route('/{id}', name: 'app_subscription_delete', methods: ['POST'])]
```
Comment (horotopia) [inline src/Controller/SubscriptionController.php:77]: Same question

File: templates/formule/_delete_form.html.twig [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<form method="post" action="{{ path('app_formule_delete', {'id': formule.id}) }}" onsubmit="return confirm('Êtes-vous certain de vouloir supprimer cette formule ?');">
+    <input type="hidden" name="_token" value="{{ csrf_token('delete' ~ formule.id) }}">
+    <button class="btn">Delete</button>
```
Comment (horotopia) [inline templates/formule/_delete_form.html.twig]:
```suggestion
    <button class="btn">Supprimer</button>
```

File: templates/subscription/index.html.twig [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+                <th>Date de fin</th>
+                <th>Actions</th>
+            </tr>
+        </thead>
+        <tbody>
+        {% for subscription in subscriptions %}
+            <tr>
+                <td>{{ subscription.id }}</td>
+                <td>{{ subscription.startDate ? subscription.startDate|date('Y-m-d') : '' }}</td>
+                <td>{{ subscription.endDate ? subscription.endDate|date('Y-m-d') : '' }}</td>
```
Comment (horotopia) [inline templates/subscription/index.html.twig]: Didn't we said d-m-Y ?

File: templates/subscription/show.html.twig [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+{% block title %}Abonnement{% endblock %}
+
+{% block body %}
+    <h1>Abonnement</h1>
+
+    <table class="table">
+        <tbody>
+            <tr>
+                <th>Date de début</th>
+                <td>{{ subscription.startDate ? subscription.startDate|date('Y-m-d') : '' }}</td>
```
Comment (horotopia) [inline templates/subscription/show.html.twig]: Same question about date

File: src/Controller/SubscriptionController.php (hunk 1 above)
Comment (Valentin460) [reply src/Controller/SubscriptionController.php:51]: Yes, I've just add a fix, companies was not visible in the dropdown list

File: src/Controller/SubscriptionController.php (hunk 2 above)
Comment (Valentin460) [reply src/Controller/SubscriptionController.php:77]: Yes also

File: templates/formule/_delete_form.html.twig (hunk 3 above)
Comment (Valentin460) [reply templates/formule/_delete_form.html.twig]: Thanks, I've forgot it

File: templates/subscription/index.html.twig (hunk 4 above)
Comment (Valentin460) [reply templates/subscription/index.html.twig]: Yes you right, I've correct that

File: templates/subscription/show.html.twig (hunk 5 above)
Comment (Valentin460) [reply templates/subscription/show.html.twig]: Same answer as last one

**Pull request merged**